Docs: Make clear the Internal Information is not made public in What's New (#409)
grafana/writers-toolkit: docs/sources/contribute-documentation/contribute-release-notes/index.md

   - **DOCUMENTATION URL (OPTIONAL)** - URL to the public documentation for this feature.
   - **ENABLEMENT VIDEO (OPTIONAL)** - Link to the video used for enablement.

>    - **INTERNAL INFORMATION (OPTIONAL)** - Information for Grafana Labs employees only. For example, ProductDNA, slack channel, FAQ, training docs or videos. Used for training and internal announcements. This will only be visible on admin.grafana.com/whats-new/, _not_ the public What's New page.

Reviewer: nit: https://developers.google.com/style/word-list#will 

```suggestion
   - **INTERNAL INFORMATION (OPTIONAL)** - Information for Grafana Labs employees only. For example, ProductDNA, slack channel, FAQ, training docs or videos. Used for training and internal announcements. It is only published as part of the post on https://admin.grafana.com/whats-new/, _not_ the public What's New page.
```
